rs/bitcoin/kyt/btc_kyt_canister.did

```diff
@@ -2,6 +2,17 @@ type tx_id = text;
 
 type bitcoin_address = text;
 
+type CheckAddressArgs = record {
+    // Bitcoin address to be checked.
+    address: bitcoin_address;
+};
+
+type CheckAddressResponse = variant { Passed; Failed };
+
 service : {
     get_inputs: (tx_id) -> (vec bitcoin_address);
+
+    // Return `Passed` if the given bitcoin address passes the KYT check, or `Failed` otherwise.
+    // May throw error (trap) if the given address is malformed or not a mainnet address.
```

Comment:
That's a good question. Why is it important to ensure that no testnet address is checked?

Reply:
> How does this work with the ckBTC test minter? Does-it not do KYT?

I think it is largely the responsibility of the caller. In the case of the ckBTC testnet minter, I don't think it needs to make calls to the KYT canister.

> Why is it important to ensure that no testnet address is checked?

It is mostly to distinguish the return result. I think it is wrong to return "Passed" when the input clearly fails outside of the function's domain. Better return an error or exception and let the caller handle it.

Reply:
>  I don't think it needs to make calls to the KYT canister.

Functionally I agree, but I do see the benefit of using the same architecture/components between test and prod, since now the KYT check via the various HTTPs outcalls may add some significant latency and the minter needs to do some additional asynchronous handling. Would it be useful to have a separate KYT instance for the ckBTC testnet minter? (probably a separate ticket)

> It is mostly to distinguish the return result. I think it is wrong to return "Passed" when the input clearly fails outside of the function's domain.

Yes I agree. With the above comment in mind, would it be useful to have some initialization arguments that's specify the network? (The OFAC checklist for testnet would simply be empty)

Reply:
> Would it be useful to have a separate KYT instance for the ckBTC testnet minter? 

Yes I agree it is useful to test the KYT workflow with testnet minter too. I created a ticket here https://dfinity.atlassian.net/browse/XC-188. I will make the network parameter configurable in that ticket too.

Reply:
Thanks for creating the ticket!